README.md

```diff
@@ -103,7 +103,7 @@ jobs:
       
       - name: Process trx reports
         id: process-trx
-        uses: im-open/process-dotnet-test-results@v2.0.2
+        uses: im-open/process-dotnet-test-results@v2.1.2
         with:
```

Comment:
This is not the right line to make this change but there is another usage of ```uses: im-open/process-dotnet-test-results@v2.0.2``` that needs to be updated to ```uses: im-open/process-dotnet-test-results@v2.1.0``` around line 85.